conda/recipes/pylibcugraph/meta.yaml

```diff
@@ -33,8 +33,10 @@ requirements:
     - ucx-proc=*=gpu
     - cudatoolkit {{ cuda_version }}.*
     - rmm {{ minor_version }}.*
+    - libraft-headers {{ minor_version }}
   run:
     - python x.x
+    - libraft-headers {{ minor_version }}
```

Comment:
FYI, `conda` supports a recipe key called `run_exports` which will ensure that a package is automatically added to the `run` dependency list if it's used in the `host` dependency list. The `run_exports` key would need to be added to the `libraft-headers` package and then it could be safely removed from the `run` list here. `libcudf` does this.

- `conda` docs - https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements
- `libcudf` example - https://github.com/rapidsai/cudf/blob/c1638869116aae2c6dde6024394279a2fb79e685/conda/recipes/libcudf/meta.yaml#L34-L35

Reply:
It looks like there's a few recipes in this PR that this comment could apply to.